source/DSCResources/DSC_WSManListener/DSC_WSManListener.psm1

```diff
@@ -51,6 +51,8 @@ function Get-TargetResource
 
     if ($listener)
     {
+        $certificate = ''
+        
```

Comment:
not sure this change is needed from looking

Reply:
`$certificate` is only used in the if block hence the move to inside it.

Reply:
I see where the certificate variable is being used in the $returnValue and would recommend on line 72 to change to `Issuer  = if ($certificate) {$certificate.Issuer} else {$null}` and remove both 54 & 55 as wouldn't be needed

Reply:
That will give a strictmode error as `$certificate` will be uninitialized.

Reply:
Sounds good to me